src/bin/statinamic-setup.js

```diff
@@ -1,12 +1,17 @@
 import npmInstallPackage from "npm-install-package"
 
-import { peerDependencies as peerDeps } from "../../package.json"
+import {
+  peerDependencies as peerDeps,
+  optionalPeerDependencies as opPeerDeps,
+} from "../../package.json"
+
+const toBeInstalledDeps = Object.assign({}, peerDeps, opPeerDeps)
```

Comment:
Can you use spread operator instead of Object.assign?

`const toBeInstalledDeps = { ...peerDeps, ...opPeerDeps }`